query-connector/src/app/query/components/selectQuery/SelectSavedQuery.tsx

```diff
@@ -0,0 +1,141 @@
+};
+
+/**
+ * Component to poulate pre-existing queries
+ * @param root0 - params
+ * @param root0.goBack - return function for the previous page
+ * @param root0.selectedQuery - specified query for future customization /
+ * application
+ * @param root0.setSelectedQuery - callback function for specified query
+ * @param root0.setShowCustomizedQuery - toggle to switch to customization
+ * view
+ * @param root0.handleSubmit - submit handler
+ * @param root0.fhirServer - fhir server to apply a query against
+ * @param root0.setFhirServer - function to update the fhir server
+ * @param root0.loadingQueryValueSets - flag for whether the value sets are
+ * still being fetched from the db
+ * @returns SelectedSavedQuery component
+ */
+const SelectSavedQuery: React.FC<SelectSavedQueryProps> = ({
+  selectedQuery,
+  fhirServer,
+  loadingQueryValueSets,
+  goBack,
+  setSelectedQuery,
+  setShowCustomizedQuery,
+  handleSubmit,
+  setFhirServer,
+}) => {
+  const [showAdvanced, setShowAdvanced] = useState(false);
+
+  return (
+    <form className="content-container-smaller-width">
+      {/* Back button */}
+      <Backlink onClick={goBack} label={"Return to select a patient"} />
+      <h1 className={`${styles.selectQueryHeaderText}`}>
+        {STEP_THREE_PAGE_TITLE}
+      </h1>
+      <div
+        className={`font-sans-md text-light ${styles.selectQueryExplanationText}`}
+      >
+        We will request all data related to your selected patient and query. By
+        only showing relevant data for your query, we decrease the burden on our
+        systems and protect patient privacy. If you would like to customize the
+        query response, click on the "customize query" button.
+      </div>
+      <h3 className="margin-bottom-3">Query</h3>
+      <div className={styles.queryRow}>
+        {/* Select a query drop down */}
+        <Select
+          id="querySelect"
+          name="query"
+          value={selectedQuery}
+          onChange={(e) => setSelectedQuery(e.target.value as USE_CASES)}
+          className={`${styles.queryDropDown}`}
+          required
+        >
+          {demoQueryOptions.map((option) => (
+            <option key={option.value} value={option.value}>
+              {option.label}
+            </option>
+          ))}
+        </Select>
+        <Button
+          type="button"
+          className={`usa-button--outline bg-white ${styles.customizeButton}`}
+          onClick={() => setShowCustomizedQuery(true)}
+        >
+          Customize query
+        </Button>
+      </div>
+
+      {showAdvanced && (
+        <div>
+          <h3 className="margin-bottom-3">Health Care Organization (HCO)</h3>
+          <Select
+            id="fhir_server"
+            name="fhir_server"
+            value={fhirServer}
+            onChange={(e) => setFhirServer(e.target.value as FHIR_SERVERS)}
+            required
+            className={`${styles.queryDropDown}`}
+          >
+            Select HCO
```

Comment:
do we want to remove this section of code if HCO has to move to SearchForm?

Reply:
I'll let Dan and Michelle way in on this but I _think_ the idea is that the HCO might be needed here in the event that we want to scope a query for the selected patient into a specific HCO. Feel free to let me know if I'm wrong though